script/git-clone-others.bat

```diff
@@ -6,7 +6,7 @@ call :initializeWorkingDirAndScriptDir
 
 rem set startDateTime=`date +%s`
 
-set gitUrlPrefix=git@github.com:droolsjbpm/
+set gitUrlPrefix=https://github.com/droolsjbpm

```

Comment:
HTTPS is now stable and even recommend by github (https://help.github.com/articles/which-remote-url-should-i-use/).